app/components/donate.jsx

```diff
@@ -0,0 +1,66 @@
+// https://github.com/EnCiv/civil-pursuit/issues/164
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import { Button } from './button'
+import cx from 'classnames'
+
+function Donate(props) {
+  const {
+    onDone = () => {}, // a function that is called when the button is clicked.  - if it exists
+    title = 'Donate to EnCiv', // text to display on hover
+    disabled = false,
+    tabIndex = 0,
+    disableOnClick = false, // if true, the button gets disabled after click and stays disabled - prevents resubmission
+    value,
+    type = 'button', // by default this is a button
+    ...otherProps
+  } = props
+
+  const classes = buttonStyles()
+
+  const handleKeyDown = e => {
+    if (e.keyCode === 32) {
+      e.stopPropagation()
+      onDone({ valid: true, value })
+    }
+  }
+
+  return (
+    <a
```

Comment:
I think the problem with spacebar not working is that an anchor and a button are being mixed.   I think for this, as much a as it would be nice to use Button, your going to need to just use an anchor and make it look like the button.

Reply:
it seems like with or without the <Button> element, pressing Enter on the donate button seems to successfully navigate to the donation page. Should I just leave it as is and assume that a user with accessibility issues know about using enter to enter hyperlinks?

It seems like in general, hyperlinks cannot be activated by a spacebar, unlike other elements (e.g. you need to press Enter on the "Terms and Conditions" hyperlink on the login page).